keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_presets.py

```diff
@@ -0,0 +1,153 @@
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(40 * alpha), kernel, 1, se_ratio, activation, 4
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(40 * alpha), kernel, 1, se_ratio, activation, 5
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(48 * alpha), kernel, 1, se_ratio, activation, 6
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(48 * alpha), kernel, 1, se_ratio, activation, 7
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(96 * alpha), kernel, 2, se_ratio, activation, 8
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(96 * alpha), kernel, 1, se_ratio, activation, 9
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(96 * alpha), kernel, 1, se_ratio, activation, 10
+    )
+    return x
+
+
+def stack_fn_l(x, kernel, activation, se_ratio, alpha=1.0):
+    x = backbone.apply_inverted_res_block(
+        x, 1, backbone.depth(16 * alpha), 3, 1, None, layers.ReLU(), 0
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 4, backbone.depth(24 * alpha), 3, 2, None, layers.ReLU(), 1
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(24 * alpha), 3, 1, None, layers.ReLU(), 2
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(40 * alpha), kernel, 2, se_ratio, layers.ReLU(), 3
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(40 * alpha), kernel, 1, se_ratio, layers.ReLU(), 4
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 3, backbone.depth(40 * alpha), kernel, 1, se_ratio, layers.ReLU(), 5
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(80 * alpha), 3, 2, None, activation, 6
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 2.5, backbone.depth(80 * alpha), 3, 1, None, activation, 7
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 2.3, backbone.depth(80 * alpha), 3, 1, None, activation, 8
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 2.3, backbone.depth(80 * alpha), 3, 1, None, activation, 9
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(112 * alpha), 3, 1, se_ratio, activation, 10
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(112 * alpha), 3, 1, se_ratio, activation, 11
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(160 * alpha), kernel, 2, se_ratio, activation, 12
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(160 * alpha), kernel, 1, se_ratio, activation, 13
+    )
+    x = backbone.apply_inverted_res_block(
+        x, 6, backbone.depth(160 * alpha), kernel, 1, se_ratio, activation, 14
+    )
+    return x
+
+
+backbone_presets_no_weights = {
+    "mobilenetv3small": {
```

Comment:
I think it's common in MobileNet literature to refer to this as mobillenetv3small-1.0, where 1.0 represents the "alpha" value

@jbischof wdyt, should we include that suffix here?

Reply:
Omitting the alpha parameter is fine with me. Users can always use a custom alpha by instantiating using the `MobileNetV3LargeBackbone` or `MobileNetV3SmallBackbone` constructors